.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+<!--
+# Pull Request Checklist
+
+* [ ] Have you read through the [contributor guidelines](./CONTRIBUTING.md)?
+* [ ] Have you signed the [Lightbend CLA](https://www.lightbend.com/contribute/cla)?
+* [ ] Have you updated the documentation?
+* [ ] Have you added tests for any changed functionality?
+-->
+## Purpose
+
+<!-- What does this PR do? -->
+
```

Comment:
From the commit comment there is normally a list of changes, where do these go?

Reply:
Added another section